### ISSUE TITLE
[Question] How to avoid transpiling node modules

### ISSUE THREAD

**Issue by vitorcamachoo** [fuse-box/fuse-box#108]
When using BabelPlugin, he will try to transpile all imported modules, including that modules from node_modules, appearing every time an error saying:
`"[BABEL] Note: The code generator has deoptimised the styling of "unknown" as it exceeds the max of "500KB"`

**Comment by nchanged** [comment]
Hm.. have you tried providing `test` option? 
Besides babel [does not go](https://github.com/fuse-box/fuse-box/blob/master/src/plugins/BabelPlugin.ts#L89) inside node_modules.

**Comment by vitorcamachoo** [comment]
This is my configuration:

```js
plugins: [
    fsbx.BabelPlugin({
			test: /\.js$/,
			config: {
				sourceMaps: true,
				presets: ["latest"],
			},
		}),
  ],
```

**Comment by nchanged** [comment]
@vitorcamachoo are you sure that it transpiles node_modules as well?

**Comment by vitorcamachoo** [comment]
If I do
```js
import angular from 'angular';
```
it will try to transpile angular.js file.

**Comment by vitorcamachoo** [comment]
I figure it out.
It was the compact property of babel that was true/auto by default, throwing the previous warning.

Thanks!

**Comment by nchanged** [comment]
Actually, i found a bug. Fixing it now

**Comment by nchanged** [comment]
Please, verify the fix in `1.3.108`
You were right, it was transpiling everything...

**Comment by vitorcamachoo** [comment]
I tested, and it's working as expected, thanks!
